### DISCUSSION TITLE
ReconFTW cannot install nuclei

### DISCUSSION THREAD

**Discussion by 00SoCRaT00** [projectdiscovery/nuclei#1378] [Q&A] [answered]

**Comment by 00SoCRaT00** [comment]
when i run as root 
GO111MODULE=on go get -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei  
i got this error 
go/pkg/mod/github.com/projectdiscovery/nuclei/v2@v2.5.5/internal/runner/update.go:27:2: //go:build comment without // +build comment
![image](https://user-images.githubusercontent.com/83826259/146257301-41799fa4-e70a-4b32-8ec2-da1f4c6cae21.png)

**Reply by 00SoCRaT00** [reply]
it is work thank you

**Comment by 00SoCRaT00** [comment]
please any one help

**Comment by forgedhallpass** [comment, marked as answer]
Try using `go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest` as the https://github.com/projectdiscovery/nuclei#install-nuclei documentation suggests.

**Reply by 00SoCRaT00** [reply]
is install in kali but not in tool reconftw 
see 
![image](https://user-images.githubusercontent.com/83826259/146259121-2f904e83-4d81-4e27-9f1b-65b4bf5baeee.png)

**Reply by forgedhallpass** [reply]
It says to install it manually, and with the command above, you just did that. It does not work automatically, because the [`install.sh`](https://github.com/six2dez/reconftw/blob/5b6266929932b59d6816c8d54550c35302df637f/install.sh#L14) script uses `go get` instead of `go install` which is deprecated in go 1.17 (requirement for the latest nuclei versions)

**Reply by 00SoCRaT00** [reply]
if i edit install.sh  file and replace install command with this 
go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest
i think it will work !
